.github/workflows/build.yml

```diff
@@ -45,7 +45,30 @@ jobs:
         with:
           python-version: ${{ matrix.python }}
 
+      - name: get-dependencies-location
+        shell: bash
+        run: |
+          SITE_PACKAGES_LOCATION=$(python -c "from distutils.sysconfig import get_python_lib; print(get_python_lib())")
+          SITE_BIN_DIR=$(python3 -c "import os; import platform; import distutils.sysconfig; pre = distutils.sysconfig.get_config_var('prefix'); bindir = os.path.join(pre, 'Scripts' if platform.system() == 'Windows' else 'bin'); print(bindir)")
+          echo "::set-output name=site_packages_loc::$SITE_PACKAGES_LOCATION"
+          echo "::set-output name=site_bin_dir::$SITE_BIN_DIR"
+          echo $SITE_PACKAGES_LOCATION
+          echo $SITE_BIN_DIR
+        id: get_dependencies
+
+      - name: Cache py-dependencies
+        id: cache-venv
```

Comment:
I think the id of this step was from when we were trying to use a virtual env, maybe just change the id to cache-dependencies